docs/fuzzing_binary-only_targets.md

```diff
@@ -302,7 +297,6 @@ some are very hard to set up...
 * S2E: [https://github.com/S2E](https://github.com/S2E)
 * TinyInst:
   [https://github.com/googleprojectzero/TinyInst](https://github.com/googleprojectzero/TinyInst)
-  (Mac/Windows only)
```

Comment:
Why is this removed?

Reply:
> There is (experimental) Linux support now! :)

_Originally posted by @ifratric in https://github.com/googleprojectzero/TinyInst/issues/67#issuecomment-1593237810_